blocks/test/serializer.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import { default as serialize } from '../serializer';
+import * as blocks from '../registration';
```

Comment:
In #355 I tried to avoid treating `registration` as some canonical set of blocks properties (since `wp.blocks` encompasses more than just registration). I don't know how you feel about this. It's a small point.